k8s/charts/openebs/templates/deployment-admission-server.yaml

```diff
@@ -8,8 +8,8 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     component: admission-webhook
-    name: admission-webhook
-    openebs.io/component-name: maya-apiserver
+    openebs.io/component-name: admission-server
```

Comment:
everyone -- admission-webhook is being used, why change to admission-server? Should we instead change the operator.yaml to have the name as admission-webhook ?

Reply:
Done